.gitignore

```diff
@@ -7,3 +7,4 @@ config.yaml
 example.db
 sendToServer.sh
 tweets-*.json
+/archived tweetsdb
```

Comment:
I don't know that I've seen a space in `gitignore` before. Is this a file with a space in the name?

Reply:
Yeah it is. I can just delete it though, it was for testing the migrate command